Add test app

Shan1024 opened the pull request:
This commit will add a new app which can be used to test fragments.

Commit:
[WSO2 Release] [Jenkins #45] [Release 1.0.0] prepare release v1.0.0

Shan1024 committed:
Add test app

This commit will add a new app which can be used to test fragments.

this commented on components/pom.xml:
```diff
@@ -15,14 +15,12 @@
 
     <groupId>org.wso2.carbon.uuf.common</groupId>
     <artifactId>components-parent</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
```
Revert all the changes in this file.

this commented on pom.xml:
```diff
@@ -15,14 +15,12 @@
   ~ limitations under the License.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Revert this change.

this commented on pom.xml:
```diff
@@ -15,14 +15,12 @@
 
     <groupId>org.wso2.carbon.uuf.common</groupId>
     <artifactId>common-parent</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
```
Revert this change.

this commented on themes/org.wso2.carbon.uuf.common.theme.default/pom.xml:
```diff
@@ -15,14 +15,12 @@
   ~ limitations under the License.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Revert all the changes in this file.

this commented on themes/pom.xml:
```diff
@@ -15,14 +15,12 @@
   ~ limitations under the License.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Revert all the changes in this file.

Shan1024 committed:
Revert "[WSO2 Release] [Jenkins #45] [Release 1.0.0] prepare release v1.0.0"

This reverts commit 5699ea9d3f84e391608adb9319e0cb6c9ffebab1.

Shan1024 committed:
Change property name

This commit will change the property name 'carbon.uuf.version'
to 'carbon-uuf-common.version'.

this commented on tests/apps/org.wso2.carbon.uuf.common.test.app/src/main/pages/index.hbs:
```diff
@@ -0,0 +1,6 @@
+{{#fillZone "content"}}
+    <h1>Hello World!</h1>
+{{/fillZone}}
+{{fragment "org.wso2.carbon.uuf.common.foundation.uuf-client"}}
```
Move this line to line no 3 (before #fillZone)

Shan1024 committed:
Change fragment location

Kishanthan commented on pom.xml:
```diff
@@ -45,6 +45,7 @@
     <modules>
         <module>components</module>
         <module>themes</module>
+        <module>tests</module>
```
if this is not used for testing purpose (i.e writing test artifacts and samples) then this module should be names as samples

Shan1024 committed:
Rename tests module to samples
